donations/apps.py

```diff
@@ -1,9 +1,7 @@
+# -*- coding: utf-8
 from django.apps import AppConfig
 
-class DonationConfig(AppConfig):
+
+class DonationsConfig(AppConfig):
     name = 'donations'
     verbose_name = "Django Donations"
-    def ready(self):
-        from .models import load_providers, load_frequencies
-        load_providers()
-        load_frequencies()
```

Comment:
Ok I've re-added to this PR, will remove it separately if agreed on a better solution.